python_modules/libraries/dagster-databricks/dagster_databricks/databricks.py

```diff
@@ -175,15 +175,23 @@ def submit_run(self, run_config, task):
         libraries = list(run_config.get("libraries", []))
         install_default_libraries = run_config.get("install_default_libraries", True)
         if install_default_libraries:
+            import dagster_databricks
+            import dagster_pyspark
```

Comment:
do these have to be deferred? these sure to be present?

Reply:
fair enough yeah these can be imported at the top